server/utils/serviceApp.js

```diff
@@ -9,6 +9,6 @@ var router = express.Router();
 
 router.use(express.static('./client'));
 router.use(express.static('./.tmp'));
-router.use(express.static('./bower_components'));
+router.use(express.static('./node_modules'));
```

Comment:
oh yeah soooo those served from a folder renamed `vendor` (also pushed to that folder)
but yeah that changed  whos da person to beep for that change?